### PR TITLE
Update readme files

### PR DESCRIPTION
**Pull request by mostafabadrshaalan**
I've updated readme files to be much better.

### CODE REVIEW

File: __leetcode.com/Easy/1-14-Longest-Common-Prefix/README.md [hunk 1]
```diff
@@ -26,3 +26,4 @@ Explanation: There is no common prefix among the input strings.
 - 0 <= strs[i].length <= 200
 - strs[i] consists of only lower-case English letters.
 
+
```
Comment (tarekbadrshalaan) [inline __leetcode.com/Easy/1-14-Longest-Common-Prefix/README.md:29]: Unnecessary update!

**Review by tarekbadrshalaan — CHANGES_REQUESTED** [review]
Hi Mostafa,
Thanks for your contribution.
I have added some comments, please consider them.

File: __leetcode.com/Easy/1-14-Longest-Common-Prefix/README.md (hunk 1 above)
Comment (mostafabadrshaalan) [reply __leetcode.com/Easy/1-14-Longest-Common-Prefix/README.md]: Done!

Comment (tarekbadrshalaan) [reply __leetcode.com/Easy/1-14-Longest-Common-Prefix/README.md]: Thanks

**Comment by tarekbadrshalaan** [comment]
Approved.
Thanks for your contribution.

**Pull request merged**